src/components/Ledger/LedgerView/LedgerGroups.js

```diff
@@ -0,0 +1,98 @@
+import React, { useCallback, useEffect, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import { withRouter } from 'react-router-dom';
+import { get } from 'lodash';
+
+import { stripesConnect } from '@folio/stripes/core';
+import { Icon } from '@folio/stripes/components';
+import { LIMIT_MAX } from '@folio/stripes-acq-components';
+
+import ConnectionListing from '../../ConnectionListing';
+import {
+  groupFundFiscalYears,
+  groupsResource,
+} from '../../../common/resources';
+
+const LedgerGroups = ({ history, funds, currency, mutator, resources }) => {
+  const openGroup = useCallback(
+    (e, group) => {
+      const path = `/finance/groups/view/${group.id}`;
+
+      history.push(path);
+    },
+    [history],
+  );
+
+  const buildQuery = useMemo(() => {
+    const fundIds = funds.map(fund => `fundId="${fund.id}"`);
+
+    return funds.length
+      ? `${fundIds.join(' or ')}`
+      : null;
+  }, [funds]);
+
+  useEffect(() => {
+    mutator.groupFundFYByFundId.reset();
+    mutator.groups.reset();
+
+    if (buildQuery) {
+      mutator.groupFundFYByFundId.GET({
+        params: {
+          limit: LIMIT_MAX,
+          query: buildQuery,
+        },
+      }).then(response => {
+        const groupIds = response.map(item => `id="${item.groupId}"`);
+        const query = response.length ? `(${groupIds.join(' or ')}) sortby name` : null;
+
+        mutator.groups.GET({ params: { query } });
+      });
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [buildQuery]);
```

Comment:
weird, why not funds here? it should be the same address in case ledger is not changed